test/renderdiff/tests/sample.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "SampleTests",                            // [required]
+    "backends": ["opengl"],                           // [optional]
+    "model_search_paths": ["third_party/models"],     // [optional] Directory from which we will obtain .glb
+						      //     files and try to match against names in the 'models' field.
+    "presets": [
+	{
+	    "name": "Standard",                       // [required]
+	    "models": ["lucy", "DamagedHelmet"],      // [optional]
```

Comment:
Maybe note that this is the basename of the file. So specifying "lucy" will search for a file named "lucy.glb".

Reply:
done